packages/amplify_authenticator/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.0.0-next.1
+
+- Minor bug fixes and improvements
```

Comment:
The authenticator does have one fix that I don't think made it into the previous release.

```suggestion
### Fixes
- fix(authenticator): remove navigator from tree for AuthenticatedState ([#2354](https://github.com/aws-amplify/amplify-flutter/pull/2354))
```

Reply:
Ah good catch 👍